### ISSUE TITLE
Commit history?

### ISSUE THREAD

**Issue by mmuman** [Nikilites/nuzu#5]
Is there any valid reason not to upload a repo with the proper history? This will make working with multiple remotes painful to whoever will want to help…

**Comment by borntohonk** [comment]
> Is there any valid reason not to upload a repo with the proper history? This will make working with multiple remotes painful to whoever will want to help…

https://www.theverge.com/2024/3/4/24090357/nintendo-yuzu-emulator-lawsuit-settlement

heres your reason.

**Comment by bigspeedfpv** [comment]
No, there isn’t. i suspect the maintainer is new to git/hub (best of luck btw) but just straight uploading all the files through the website is wild

**Comment by Stay1444** [comment]
Yeah, I'm also searching for yuzu's mbedtls fork, because I have a clone of that but it seems  that it isn't the latest version that they used since I'm getting a link error. I asked how did they build it but apparently they just reuploaded existing binaries.
